package.json

```diff
@@ -9,6 +9,7 @@
     "@types/request": "^2.0.3",
     "@types/run-sequence": "0.0.29",
     "@types/supertest": "^2.0.3",
+    "child_process": "^1.0.2",
```

Comment:
Even If we have to child_processs, do we really need this? The child_process modules is built-in library as far as I know. WDYT?

Reply:
If we can use the built-in library, it's a good idea to use it. I'll look into it too. :)

Reply:
But if you use gulp.shell, we might not need it :)